tests/test_config.yml

```diff
@@ -15,30 +15,25 @@ snapshot:
     password: min0s
     host: localhost
     port: 5432
+broker:
+  host: localhost
+  port: 9092
+  queue:
+    database: order_db
+    user: minos
+    password: min0s
+    host: localhost
+    port: 5432
+    records: 10
+    retry: 2
 events:
     broker: localhost
     port: 9092
```

Comment:
In this case, `events.broker` and `events.port` must be also removed as they are already defined at `broker.host` and `broker.port`